permissions/views.py

```diff
@@ -0,0 +1,204 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import User
+from django.http import JsonResponse
+from django.shortcuts import redirect, render, get_object_or_404
+from django.views.decorators.http import require_POST
+
+from yaksh.models import Course, Quiz, Lesson
+
+from .models import Permission, Role, Team
+from .utils import format_perm
+
+
+# Create your views here.
+
+
+@login_required
+def home(request):
+    user = request.user
+    # Get users
+    users = User.objects.all().exclude(username=user.username)
+    teams = user.team_members.all()
+    courses = Course.objects.filter(creator=user)
+
+    context = {
+        "users": users,
+        "teams": teams,
+        "courses": courses
+    }
+
+    return render(request, "home.html", context)
+
+
+@require_POST
+@login_required
+def create_team(request):
+    user = request.user
+    team_name = request.POST.get("team_name")
+    members_list = request.POST.getlist("members")
+    courses_list = request.POST.getlist("courses")
+
+    if len(team_name) > 0:
+        team = Team.objects.create(
+            name=team_name,
+            created_by=user
+        )
+
+        members = User.objects.filter(username__in=members_list)
+        team.members.add(*members)
+        team.members.add(user)
+
+        courses = Course.objects.filter(id__in=courses_list)
+        team.courses.add(*courses)
+
+        team.save()
```

Comment:
Do you need this? Just doing `add` should save the data in the database.

Reply:
Yeah, just `add` saves it to the db.